test/validation-ui-tests.js

```diff
@@ -625,6 +625,74 @@ QUnit.test('Issue #44 - Validation Element - Is Invalid Test', function(assert)
 
 });
 
+QUnit.test('Validatable can become non-validatable', 0, function(assert) {
```

Comment:
It was an accident leaving this test behind. Removing it as redundant and incomplete.